Packs/SymantecBlueCoatProxySG/README.md

```diff
@@ -0,0 +1,34 @@
+# Symantec Bluecoat ProxySG
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Bluecoat ProxySG to forward Syslog messages with the access log 'main' format.
+
+Go to the ProxySG software and open the "Configuration" tab [Product Doc](https://knowledge.broadcom.com/external/article/166529/sending-access-logs-to-a-syslog-server.html);
+1. From the left sidebar, navigate to **Access Logging** > **Logs** > **Logs** tab, and create a new log with the 'main' format defined for it.
+2.  Click the **Upload Client** tab and configure your log as follows:
+
+   | Field | Value |
+   | --- | --- |
+   | Client type | Custom Client |
+   | Save the log file as | text file |
+3. In the **Custom Client Settings** dialog box, point the Custom Client to the broker, specifying the IP address and port number.
```

Comment:
How do you access this?

Reply:
It wasn't specifically mentioned in the product doc, I believe it may be a popup when choosing the settings for the Custom Client option.